libvirt/tests/src/virtual_network/qemu/netperf.py

```
@@ -0,0 +1,863 @@
+            client_path,
+            server,
+            int(l) * 1.5,
+            nf_args,
+        )
+        option += " >> %s" % fname
+        netperf_base.netperf_thread(params, numa_enable, client_s, option, fname)
+
+    def all_clients_up():
+        try:
+            content = netperf_base.ssh_cmd(clients[-1], "cat %s" % fname)
+        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)